src/reviews/views.py

```diff
@@ -327,3 +332,77 @@ def get_success_url(self):
         if query_string:
             return url + '?' + query_string
         return url
+
+
+def review_stages(request):
+    current_review_stages_setting = {}
+    review_stages_list = [
+        'Call for Proposals',
+        'Locked (proposal editing and reviewing disabled)',
+        'First Round Review', 'Modification Stage', 'Second Round Review',
+        'Internal Decision', 'Announcement of Acceptance'
+    ]
+    review_stages_var = [
+        'proposals.creatable', 'proposals.editable', 'proposals.withdrawable',
+        'reviews.visible.to.submitters', 'reviews.stage',
+        'proposals.disable.after'
+    ]
+
+    # Initialize current setting with existing value
+    for tag in review_stages_var:
+        key = settings.CONFERENCE_DEFAULT_SLUG + '.' + tag
+        value = reg.get(key, '')
+        update_current_review_stages_setting(tag, value, current_review_stages_setting)
+
+    if request.method == 'POST':
+        for tag in review_stages_var:
+            key = settings.CONFERENCE_DEFAULT_SLUG + '.' + tag
+            if tag == 'proposals.disable.after':
+                if request.POST[tag] == "":
+                    continue
+                else:
+                    date_time_obj = date_preprocess(DATETIME_INPUT_FORMATS, request.POST[tag])
+                    if date_time_obj is None:
+                        messages.error(request, 'Please input valid date format : " + "%Y-%m-%dT%H:%M')
+                        value = None
```

Comment:
why save None on error? maybe we can raise exception and respond with 400 instead?